Tasks/DeployVisualStudioTestAgent/TestAgentConfiguration.ps1

```diff
@@ -842,6 +849,7 @@ function InvokeTestAgentConfigExe([string[]] $Arguments, [string] $Version, [Sys
         return $out
     }
 
+    Write-Host "##vso[task.logissue type=error;code=001003;]"
     throw "Did not find TestAgentConfig.exe at : $exePath. Ensure that TestAgent is installed."
 }
```

Comment:
line number 903 also has a throw statement ... we should log an error there as well